pystiche_papers/johnson_alahi_li_2016/_loss.py

```diff
@@ -38,6 +38,25 @@ def content_loss(
     layer: str = "relu2_2",
     score_weight: Optional[float] = None,
 ) -> ops.FeatureReconstructionOperator:
+    r"""Content_loss from "Perceptual Losses for Real-Time Style Transfer and
+    Super-Resolution" by Johnson, Alahi, and Li (original authors) :cite:`JAL2016`.
```

Comment:
I wonder if we can shorten this somehow. I don't think we need this block at the top of every docstring. Maybe "Content loss from :cite:`JAL2016`"?

Reply:
I think you are right and for the docstrings this shortened form is sufficient. But is it also possible to integrate the complete form with links to the original paper and the original github repository above the single docstrings for pystiche_papers.johnson_alahi_li_2016 in general?

Reply:
Sure. I'll add an example for this later.